dice-game-ui-angular/src/app/nav-menu/nav-menu.component.ts

```diff
@@ -1,13 +1,18 @@
-import { Component } from '@angular/core';
```

Comment:
Changes in this file are not needed at all.

Reply:
Removed

Reply:
Not removed.